kubernetes.go

```diff
@@ -83,8 +89,7 @@ func (kd *KubernetesDiscoverer) GetDestinationsForService(serviceName string) ([
 			continue
 		}
 
-		// prepend with // so that it is a valid URL parseable by url.Parse
-		podIp := fmt.Sprintf("http://%s:%s", pod.Status.PodIP, forwardPort)
+		podIp := fmt.Sprintf("%s:%s", pod.Status.PodIP, forwardPort)
```

Comment:
Can you also add a test to verify the assertion that "This will still work with forwarding with http because in proxy.go, the doPost method will check and append it if its missing."?

Reply:
added back the "http://" prefix for tcp and http ports. Pulled out some of this logic into another function and added some tests around that.